### PR TITLE
Documentation SE : OIDC Postman extension

### PR DESCRIPTION
**Pull request by tvallin**
This is additional content on OIDC security guide to check how to get token based on different flows with Postman. 

Signed-off-by: tvallin <thibault.vallin@oracle.com>

### CODE REVIEW

**Comment by tvallin** [comment]
If these modifications are satisfying, I will update Helidon MP documentation as well.

**Review by tomas-langer — COMMENTED** [review]
Looks good from technical pov. @ljamen please review from docs pov. Thanks!

File: docs/se/guides/05_security_oidc.adoc [hunk 1]
```diff
@@ -348,6 +357,81 @@ now protected, its access is limited, and the tests are not built to take oidc s
...
+----
+
+Do not forget to replace the `client secret` by its value (generated during Create a Client), and `authorization code`
+by the code value in the query parameter. Send the request by pressing `Send`. Keycloak returns an access token and
+a refresh token.
+
+==== Resource Owner Password Credentials Grant (Direct Access Grants)
+
+The Direct Access Grants flow is used by REST clients which wants to request tokens on behalf of a user.
+Use Postman to make this request on behalf of `myuser`. Select the GET method and enter this url:
```
Comment (ljamen) [inline docs/se/guides/05_security_oidc.adoc:412]: To use Postman to make this request on behalf of `myuser`, select the GET method and enter this URL:

File: docs/se/guides/05_security_oidc.adoc [hunk 2]
```diff
@@ -348,6 +357,81 @@ now protected, its access is limited, and the tests are not built to take oidc s
...
+`http://localhost:7987/greet/`. Under `Authorization` tab, select authorization type`OAuth 2.0`. Under it, complete the
+sentence  `Add authorization data to` with `Request Headers`, and fulfil the required fields.
```
Comment (ljamen) [inline docs/se/guides/05_security_oidc.adoc:414]: and complete the required fields.

File: docs/se/guides/05_security_oidc.adoc [hunk 3]
```diff
@@ -348,6 +357,81 @@ now protected, its access is limited, and the tests are not built to take oidc s
...
+{"key":"Access Token URL","value":"http://localhost:8080/auth/realms/myRealm/protocol/openid-connect/token"},
+{"key":"Client ID","value":"myClientID"},
+{"key":"Client Secret","value":"client secret"},
+{"key":"Username","value":"myuser"},
+{"key":"Password","value":"password"},
+{"key":"Scope","value":"openid"},
+{"key":"Client Authentication","value":"Send as Basic Auth Header"}]
+----
+
+Again, make sure to replace `client secret` by the actual client secret. Click on `Get New Access Token`. A pop up
```
Comment (ljamen) [inline docs/se/guides/05_security_oidc.adoc:430]: A popup window

File: docs/se/guides/05_security_oidc.adoc [hunk 4]
```diff
@@ -348,6 +357,81 @@ now protected, its access is limited, and the tests are not built to take oidc s
...
+window appears with Authentication complete, click on proceed to display access, refresh and identity token.
+Copy past the access token to `Access Token` field and press `Send`. Helidon greeting application sends back
```
Comment (ljamen) [inline docs/se/guides/05_security_oidc.adoc:432]: Copy and paste the access token

**Comment by tomas-langer** [comment]
@tvallin - could you please mark fixed issues as resolved and re-request review?
Thanks

File: docs/se/guides/05_security_oidc.adoc [hunk 5]
```diff
@@ -348,6 +357,81 @@ now protected, its access is limited, and the tests are not built to take oidc s
...
+=== Test Keycloak process with Postman
+
+Keycloak supports many authentication and authorization flows, but only two of them will be shown. This section
+describes another way you can get an access token or refresh a token or identity token. The identity token contains
+information about the user. The access token contains access information that the application can use to determine what
+resources the user is allowed to access. Once expired, the refresh token allows the application to obtain a new access
+token. As these tokens contain sensitive information, they are valid for a very short period. It is possible to make them
+last longer in order to let you manipulate them with Postman. To do so:
+
+* Open the Postman Console.
```
Comment (ljamen) [inline docs/se/guides/05_security_oidc.adoc:369]: Numbered list

**Review by ljamen — APPROVED** [review]
Looks good - just a small suggestion to change the bulleted list to a numbered list.

**Review by tomas-langer — APPROVED** [review]
LGTM

**Pull request merged**